lib/dqt/api.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Dqt
+  class Api
+    delegate :dqt_record, to: :v1
+
+    def initialize(client:)
+      self.client = client
+      self.v1 = nil
+    end
+
+    def v1
+      @v1 ||= V1.new(client: client) # rubocop:disable Naming/VariableNumber
+    end
+
+  private
+
+    attr_accessor :client
+    attr_writer :v1
+  end
```

Comment:
What's the purpose of this class? Do we expect different versions of api client? Do we expect to need them in our codebase at the same time? Is now the time to add support for it?

Reply:
🤷 Copied from Claim. Check commitwise for what I actually changed. It seemed prudent to use their client, as they wrote the API

Reply:
If they have versioned the client, I expect they plan to version the API